readthedocs/projects/querysets.py

```diff
@@ -54,6 +56,24 @@ def private(self, user=None):
             return self._add_user_repos(queryset, user)
         return queryset
 
+    def is_active(self, project):
+        """
+        Check if the project is active.
+
+        The check consists on,
+          * the Project shouldn't be marked as skipped.
```

Comment:
not sure I follow this sentence

Reply:
:disappointed: 

I just want to say that it checks for `project.skip`. That's why the project must not has to be marked as skipped... :)